code/datums/status_effects/neutral.dm

```diff
@@ -45,3 +45,11 @@
 
 /datum/status_effect/high_five/on_remove()
 	owner.visible_message("[owner] was left hanging....")
+
+/datum/status_effect/conversion_cooldown
+	id = "conversion_cooldown"
+	duration = 20 SECONDS
+	alert_type = null
```

Comment:
this should have a define in `status_effects.dm` for consistency with all other status effects.